bin/node/runtime/src/lib.rs

```diff
@@ -1282,6 +1283,22 @@ mod mmr {
 	pub type Hashing = <Runtime as pallet_mmr::Config>::Hashing;
 }
 
+/// There are two extensions returning the priority:
+/// 1. The `CheckWeight` extension.
+/// 2. The `TransactionPayment` extension.
+///
+/// The first one gives a significant bump to `Operational` transactions, but for `Normal`
+/// it's within `[0..MAXIMUM_BLOCK_WEIGHT]` range.
+///
+/// The second one roughly represents the amount of fees being paid (and the tip) with
+/// size-adjustment coefficient. I.e. we are interested to maximize `fee/consumed_weight` or
+/// `fee/size_limit`. The returned value is potentially unbounded though.
+///
+/// The idea for the adjustment is scale the priority coming from `CheckWeight` for
+/// `Normal` transactions down to zero, leaving the priority bump for `Operational` and
+/// `Mandatory` though.
+const CHECK_WEIGHT_PRIORITY_DIVISOR: TransactionPriority = MAXIMUM_BLOCK_WEIGHT;
```

Comment:
shouldn't this be `MAXIMUM_BLOCK_WEIGHT / 10`?

Reply:
It used to be, @kianenigma suggested to reduce it down to 0 to avoid any effects.